From 0e923dc82cbd0376ad7620a5a8f7696bf30d0d7d Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Tue, 23 Sep 2014 10:25:01 +0800
Subject: [PATCH 0549/5242] MLK-10008 ARM: imx6qdl: Enable pcie on imx6qdl
 sabreauto

commit  2fd334a7a78433493db62a147a6904a7b91f02e9 from
https://source.codeaurora.org/external/imx/linux-imx.git

- enable pcie on imx6qdl sabreauto boards.

Signed-off-by: Richard Zhu <richard.zhu@freescale.com>
(cherry picked from commit 21c4323d54ec3a25bb73deb98d23ddd1bb3c6c03)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index 3620064..4a2fbd3 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -979,6 +979,10 @@
 	status = "okay";
 };
 
+&pcie {
+	status = "okay";
+};
+
 &spdif {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_spdif>;
-- 
1.7.9.5

